docs/configuration.md

```diff
@@ -1102,6 +1102,27 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.enabled</code></td>
+  <td>true</td>
+  <td>
+    To determine whether external shuffle uses db to store shuffle state.
+    In standalone and Yarn modes, for more detail, see
+    <a href="spark-standalone.html#Cluster Launch Scripts">this description</a>.
```

Comment:
Should also link the detailed page for Yarn?

Reply:
No, yarn always has this behavior `spark.shuffle.service.db.enabled` enabled, I will remove `and Yarn modes`

Reply:
done